.github/workflows/_release_please.yaml

```
@@ -0,0 +1,24 @@
+name: Release Please
+
+on:
+  workflow_call:
+    secrets:
+      release_please_token:
+        required: true
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub App Token
+        id: app-token
+        uses: actions/create-github-app-token@v1.11.0
+        with:
+          app-id: 1077859
+          private-key: ${{ secrets.release_please_token }}
+
+      - uses: googleapis/release-please-action@v4
```

Can we use a fully qualified version here?
```suggestion
      - uses: googleapis/release-please-action@v4.1.3
```